### PR TITLE
869 updating register and verification workflow documentation

### LINKED ISSUE #869: Update Register documentation with the new workflow
register no longer shows a "success view", now it follows a workflow depending on other variables such as AccountStatus and autoLogin
here is the spec
https://github.com/stormpath/stormpath-framework-spec/blob/master/registration.md

### PR DESCRIPTION
**Pull request by mzumbadoate**
resolves #869

### CODE REVIEW

File: docs/source/registration.rst [hunk 1]
```diff
@@ -184,15 +189,25 @@ You can change the value to reflect a different path if you wish.
...
+
+            stormpath.web.verifyEmail.autoLogin = false
+            stormpath.web.verifyEmail.nextUri = /login?status=verified
+
+        As you can see, this URI has a ``status=verified`` query parameter.  The plugin's default login view will recognize the query parameter and show the user a nice message explaining that their account has been verified and that they can log in:
+
+        .. image:: /_static/login-verified.png
 
-    stormpath.web.verifyEmail.nextUri = /login?status=verified
+    * If ``autoLogin`` is true, the verification workflow will log in the user and redirect to login 'next' URI
```
Comment (mraible) [inline docs/source/registration.rst]: I'd reword this as:

If ``autoLogin`` is true, the verification workflow will login the user and redirect to the ``nextUri`` value.

File: extensions/spring/boot/docs/source/registration.rst [hunk 2]
```diff
@@ -204,15 +209,25 @@ You can change the value to reflect a different path if you wish.
...
+
+            stormpath.web.verifyEmail.autoLogin = false
+            stormpath.web.verifyEmail.nextUri = /login?status=verified
+
+        As you can see, this URI has a ``status=verified`` query parameter.  The plugin's default login view will recognize the query parameter and show the user a nice message explaining that their account has been verified and that they can log in:
+
+        .. image:: /_static/login-verified.png
 
-    stormpath.web.verifyEmail.nextUri = /login?status=verified
+    * If ``autoLogin`` is true, the verification workflow will log in the user and redirect to login 'next' URI
```
Comment (mraible) [inline extensions/spring/boot/docs/source/registration.rst]: Same comment as above for `docs/source/registration.rst`.

**Comment by mraible** [comment]
@mzumbadoate A couple minor comments about wording. Back to you.

**Comment by mzumbadoate** [comment]
@mraible fixed

**Comment by mraible** [comment]
LGTM 👍 if CI passes
